tests/testthat/test-BinarySegmentationTests.R

```diff
@@ -12,33 +12,52 @@ set.seed(1)
 test_that(desc="Binary Segmentation + Change in mean: Test 1",{
   data  <-  c(rnorm(10, 100, 10), rnorm(10, 50, 10))
   ans <- BinSeg::binseg(data, "BS", "mean_norm", 1, 1)
-  expect_equal(ans[["cpts"]], 9)
+  check_ans <- changepoint::cpt.mean(data=data, penalty="None", method="BinSeg", Q=1, test.stat="Normal")@cpts[1]
+  expect_equal(ans[["cpts"]], check_ans)
 })
 
 
 test_that(desc="Binary Segmentation + Change in mean and variace: Test 1", {
```

Comment:
typo: variance.
instead of Test 1, Test 2, etc, can you describe the input and output in words?
for example "single changepoint with meanvar_norm model using random normal data should be same as changepoint package"